graphql_api/mutation/project.py

```diff
@@ -36,6 +48,58 @@ def mutate(self, info, name: str, description: Optional[str] = None):
         return CreateProject(project=project, ok=True)
 
 
+class CreateProjectByWorkflowStore(graphene.Mutation):
+    class Arguments:
+        name = graphene.String(required=True)
+        description = graphene.String(required=True)
+        tokenizer = graphene.String(required=True)
+        storeId = graphene.String(required=True)
+
+    ok = graphene.Boolean()
+    project = graphene.Field(lambda: Project)
+
+    def mutate(self, info, name: str, description: str, tokenizer: str, storeId: str):
+        workflow_postgres_url = os.getenv("WORKFLOW_POSTGRES")
+        user = auth.get_user_by_info(info)
+        organization = auth_manager.get_organization_by_user_id(user.id)
+
+        if not bucket_exists(str(organization.id)):
+            create_bucket(str(organization.id))
+
+        project = project_manager.create_project(
+            organization.id, name, description, user.id
+        )
+        project_manager.update_project(project_id=project.id, tokenizer=tokenizer)
+        engine = create_engine(workflow_postgres_url)
+
+        Session = sessionmaker(engine)
+        with Session() as session:
+            results = session.execute(
+                f"SELECT record FROM store_entry WHERE store_id = '{storeId}'"
+            ).all()
```

Comment:
not sure regarding the approach here. This will create an engine every single time which shouldn't be done with more than once per database URL -> [doc](https://docs.sqlalchemy.org/en/13/core/connections.html#basic-usage) .

A singletone would do the trick imo :) 


also with this we now have database knowledge here from the workflow app which links them quite closely. 
I think we could do it similar to uploading stuff in general => create a task and upload to minio (from workflow) which then triggers the usual routine.

Reply:
we had the discussion with minio, but workflow is meant to be coupled super closely to refinery. So it makes perfect sense to use the direct db connection, as workflow is the data-sender for refinery.

Not an expert with the sessionmaker, but a singleton I can build :)

Reply:
done.